charts/tidb-cluster/values.yaml

```diff
@@ -398,13 +398,17 @@ binlog:
     # or to arbitrary policies determined by the cluster administrators.
     # refer to https://kubernetes.io/docs/concepts/storage/storage-classes
     storageClassName: local-storage
-    storage: 20Gi
-    syncLog: true
-    # a integer value to control expiry date of the binlog data, indicates for how long (in days) the binlog data would be stored.
-    # must bigger than 0
-    gc: 7
-    # number of seconds between heartbeat ticks (in 2 seconds)
-    heartbeatInterval: 2
+    storage: 100Gi
+    # Please refer to https://github.com/pingcap/tidb-binlog/blob/master/cmd/pump/pump.toml for the default
+    # pump configurations (change to the tags of your pump version),
+    # just follow the format in the file and configure in the 'config' section
+    # as below if you want to customize any configuration.
+    config: |
+      gc = 7
+      heartbeat-interval = 2
+      [storage]
+      sync-log = true
+      stop-write-at-available-space = "100 gib"
```

Comment:
```
# stop write when disk available space less then the configured size
# 42 MB -> 42000000, 42 mib -> 44040192
# default: 10 gib
# stop-write-at-available-space = "10 gib"
```
I think it should be 1 Gigabytes or less

Reply:
Yes, this is available space size, using default.